multiplexer/src/index.ts

```diff
@@ -525,13 +512,14 @@ Agent: ${encodeHashToBase64(cellId[1])}`;
     doError(res, e);
     return;
   }
-});
+});redirecting
```

Comment:
This is probably a typo

```suggestion
});
```